apps/next/.env.example

```diff
@@ -26,4 +26,9 @@ CLOUD_BUILDER_LONG_NAME=x
 CLOUD_BUILDER_NAME=x
 DOCKERHUB_ORG=x
 DOCKERHUB_REPO=x
-LOOPS_WAITLIST_FORM_URL=x
\ No newline at end of file
+
+# Use these to disable telemetry for Clerk.
+CLERK_TELEMETRY_DISABLED=1
+NEXT_PUBLIC_CLERK_TELEMETRY_DISABLED=1
```

Comment:
nit but can you move these up under NEXT_TELEMETRY_DISABLED=1

Reply:
Done.